jsr352/core/src/test/java/org/hibernate/search/jsr352/massindexing/MassIndexingJobWithMultiTenancyIT.java

```diff
@@ -0,0 +1,123 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.jsr352.massindexing;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import javax.batch.operations.JobOperator;
+import javax.batch.runtime.BatchStatus;
+import javax.batch.runtime.JobExecution;
+import javax.persistence.EntityManagerFactory;
+
+import org.hibernate.Session;
+import org.hibernate.dialect.Dialect;
+import org.hibernate.dialect.H2Dialect;
+import org.hibernate.search.FullTextSession;
+import org.hibernate.search.Search;
+import org.hibernate.search.jsr352.massindexing.test.entity.Company;
+import org.hibernate.search.jsr352.test.util.JobFactory;
+import org.hibernate.search.jsr352.test.util.JobTestUtil;
+import org.hibernate.search.test.SearchTestBase;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import org.fest.util.Collections;
+
+import static org.fest.assertions.Assertions.assertThat;
+
+/**
+ * @author Mincong Huang
+ */
+public class MassIndexingJobWithMultiTenancyIT extends SearchTestBase {
+
+	public static final Dialect DIALECT = new H2Dialect();
```

Comment:
Yes, you're right. Actually, I deleted it once and then job failed, but it must be due to another cause which I didn't find out. It has been removed now.